test/temple.js

```diff
@@ -38,15 +38,19 @@ describe('Banner', function () {
     });
   });
 
-  
+
 
   /**
    * GET_TERMS()
    */
   describe('#getTerms()', function () {
 
     before(async () => {
-      cache.terms = await banner.getTerms();
+      try {
+        cache.terms = await banner.getTerms();
+      } catch (e) {
+        console.error(e);
+      }
     });
```

Comment:
Using a try/catch here won't cause Mocha to report the test suite failing. We also shouldn't be using try/catch here anyways since the function isn't supposed to throw an error. The caching feature should probably integrated with the first unit test.

The same applies to the other unit tests in this file.